config/frr-on-ci/bgpTypeFRR.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
+  namespace: system
+spec:
+  template:
+    spec:
+      containers:
+        - name: manager
+          env:
+            - name: METALLB_BGP_TYPE
+              value: "frr"
```

Comment:
we don't want to test native E2E?

Reply:
I don't think we care too much about that, rather we want to verify that some tests pass using the Operator's CRDs.
Since FRR covers more options than the native (like BFDProfiles) I defaulted to it.